docs/telemetry/README.md

```diff
@@ -199,7 +202,8 @@ If it is set in the diagram, we also add target engine profile information:
 ```json
 {
   "engineProfile": {
-    "executionPlatform": "Camunda Cloud"
+    "executionPlatform": "Camunda Cloud",
+    "executionPlatformVersion": "1.1"
```

Comment:
This was added to a wrong event. Should be added after line 65.

Reply:
Thank you for the heads up! I moved this to the proper place.